Cargo.toml

```diff
@@ -4,6 +4,8 @@ members = [
     'pallets/poa',
     'pallets/poa/rpc',
     'pallets/token_migration',
+    'pallets/democracy',
+    'pallets/simple_democracy',
```

Comment:
What is the difference between the two pallets?

Reply:
Reg. `simple_democracy`, quoting from the pallet here https://github.com/docknetwork/dock-substrate/pull/91/files#diff-da75ce70c77bb2582d4ea0e47c35f912a4ca54d55b961d44b92f08d8e1d7ea3eR1

> This is a facade over Substrate's democracy pallet.

Will add more details in the pallet documentation.